pkg/logging/http_access.go

```diff
@@ -49,7 +55,21 @@ func NewHTTPAccessLogHandler(logger Logger, level logrus.Level, message string)
 			if v := r.Header.Get("X-Real-Ip"); v != "" {
 				fields["x-real-ip"] = v
 			}
-			logger.WithFields(fields).Log(level, message)
+			logger.WithFields(fields).Log(rl.level, message)
+		})
+	}
+}
+
+// SerAccessLogLevelHandler overrides the log level set in
+// NewHTTPAccessLogHandler for a specific endpoint. Use log level 0 to suppress
+// log messages.
+func SerAccessLogLevelHandler(level logrus.Level) func(h http.Handler) http.Handler {
```

Comment:
Ser=Set?

Reply:
Huh, thanks. :)

Reply:
I thought it was short for `Service` whatever `ServiceAccessLogLevelHandler` means...